addons/stock/models/stock_picking.py

```diff
@@ -2083,6 +2083,16 @@ def _can_return(self):
         self.ensure_one()
         return self.state == 'done'
 
+    def _add_reference(self, reference=False):
```

Comment:
```suggestion
    def _add_reference(self, reference):
```
Don't think `reference` should be optional, as it would lead to a traceback when accessing `False.id`